sdk/loadtestservice/azure-analytics-loadtestservice/tests/test_smoke_async.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from testcase import LoadtestservicePowerShellPreparer
+from testcase_async import LoadtestserviceAsyncTest
+
+
+class LoadtestserviceSmokeAsyncTest(LoadtestserviceAsyncTest):
+
+    @LoadtestservicePowerShellPreparer()
+    async def test_smoke_create_or_update_test(self, loadtestservice_endpoint):
+        client = self.create_client(loadtestservice_endpoint)
+        test_id="d7c9990e"  
+        body_test={"resourceId":"/subs","testId":"d7c9990e","description":"","displayName":"newtest","loadTestConfig":{"engineSize":"m","engineInstances":1,"splitAllCSVs":False},"secrets":{},"environmentVariables":{},"passFailCriteria":{"passFailMetrics":{}},"keyvaultReferenceIdentityType":"SystemAssigned","keyvaultReferenceIdentityId":None}
```

Comment:
@Yashikaj14 please format the code with IDE. It is hard to read if this line is too long